pkg/middleware/auth.go

```diff
@@ -47,7 +47,8 @@ func notAuthorized(c *m.ReqContext) {
 		return
 	}
 
-	c.SetCookie("redirect_to", url.QueryEscape(setting.AppSubUrl+c.Req.RequestURI), 0, setting.AppSubUrl+"/", nil, false, true)
+	c.Resp.Header().Del("Set-Cookie")
```

Comment:
Why wasn't this needed before, or was it?

Reply:
I think it is not needed.
It was introduced by this [commit](https://github.com/grafana/grafana/pull/19787/commits/f80d23e83617e3607b9a4961e676c6b337f282ba) and I hesitated to delete it because I was not sure.